packages/swap/src/routes/__tests__/v2/__snapshots__/quote.test.ts.snap

```diff
@@ -1,5 +1,207 @@
+      "swap": 12,
+    },
+    "estimatedPrice": "0.000384",
+    "includedFees": [
+      {
+        "amount": "500000000",
+        "asset": "USDC",
+        "chain": "Ethereum",
+        "type": "NETWORK",
+      },
+    ],
+    "isOnChain": true,
+    "poolInfo": [
+      {
+        "baseAsset": {
+          "asset": "ETH",
+          "chain": "Ethereum",
+        },
+        "fee": {
+          "amount": "2000000000",
+          "asset": "USDC",
+          "chain": "Ethereum",
+        },
+        "quoteAsset": {
+          "asset": "USDC",
+          "chain": "Ethereum",
+        },
+      },
+    ],
+    "recommendedSlippageTolerancePercent": 1,
+    "srcAsset": {
+      "asset": "USDC",
+      "chain": "Ethereum",
+    },
+    "type": "REGULAR",
+  },
+]
+`;
+
+exports[`server > GET /v2/quote > on chain > properly quotes on chain swaps between stables 1`] = `
+[
+  {
+    "depositAmount": "1000000000000",
+    "destAsset": {
+      "asset": "USDC",
+      "chain": "Ethereum",
+    },
+    "egressAmount": "1000000000000",
+    "estimatedDurationSeconds": 12,
+    "estimatedDurationsSeconds": {
+      "swap": 12,
+    },
+    "estimatedPrice": "1",
+    "includedFees": [
+      {
+        "amount": "100000000",
+        "asset": "USDC",
+        "chain": "Ethereum",
+        "type": "NETWORK",
+      },
+    ],
+    "isOnChain": true,
+    "poolInfo": [
+      {
+        "baseAsset": {
+          "asset": "USDC",
+          "chain": "Solana",
+        },
+        "fee": {
+          "amount": "1000000000",
+          "asset": "USDC",
+          "chain": "Solana",
+        },
+        "quoteAsset": {
+          "asset": "USDC",
+          "chain": "Ethereum",
+        },
+      },
+    ],
+    "recommendedSlippageTolerancePercent": 0.5,
+    "srcAsset": {
+      "asset": "USDC",
+      "chain": "Solana",
+    },
+    "type": "REGULAR",
+  },
+]
+`;
+
+exports[`server > GET /v2/quote > on chain > respects the minimum network fee 1`] = `
+[
+  {
+    "depositAmount": "1000000000",
+    "destAsset": {
+      "asset": "USDC",
+      "chain": "Ethereum",
+    },
+    "egressAmount": "1000000000",
+    "estimatedDurationSeconds": 12,
+    "estimatedDurationsSeconds": {
+      "swap": 12,
+    },
+    "estimatedPrice": "1",
+    "includedFees": [
+      {
+        "amount": "500000",
+        "asset": "USDC",
+        "chain": "Ethereum",
+        "type": "NETWORK",
+      },
+    ],
+    "isOnChain": true,
+    "poolInfo": [
+      {
+        "baseAsset": {
+          "asset": "USDC",
+          "chain": "Solana",
+        },
+        "fee": {
+          "amount": "1000000",
+          "asset": "USDC",
+          "chain": "Solana",
+        },
+        "quoteAsset": {
+          "asset": "USDC",
+          "chain": "Ethereum",
+        },
+      },
+    ],
+    "recommendedSlippageTolerancePercent": 0.5,
+    "srcAsset": {
+      "asset": "USDC",
+      "chain": "Solana",
+    },
+    "type": "REGULAR",
+  },
+]
+`;
+
```

Comment:
can we get a test for onChainSwaps with DCA? Asking because i don't think it will adjust duration according to chunks.

Reply:
changed one of the tests, everything works as expected

Reply:
🙏